documentation/book/api/snippet/GET/http-response.adoc

```diff
@@ -0,0 +1,9 @@
+==== Example HTTP response
+
+===== Response 200
+[source,json]
+----
+{
+  "bridge_version" : "0.16.0"
```

Comment:
Why 0.16.0?

Reply:
It's a very old example :-D I will update to the last one even if I think we can't get it automatically so it will stay at 0.30.0 for years again :-P

Reply:
I guess it should be removed, or have some placeholder or something if it cannot be kept up-to-date automatcially.

Reply:
I made it automatic, I was able to use the `{ProductVersion}` variable we pass on the command line already (not sure why because it wasn't used elsewhere, but was useful for this case :-)). 
Now getting something like this when building the snapshot version:

![image](https://github.com/user-attachments/assets/a0d86b36-7f54-46a7-b848-1c4c2f5863df)